frontend/src/components/FriendGameRecords/FriendOutcome.css

```
@@ -1,58 +1,62 @@
-.game-records .friend-one-game-record .outcome {
-	display: flex;
-	height: 45px;
-	align-items: center;
-	font-family: 'DungGeunMo';
+.friendoutcome {
+  display: flex;
+  height: 40px;
+  align-items: center;
+  font-family: 'DungGeunMo';
 }
 
-.game-records .friend-one-game-record .profile-container {
-	display: flex;
-    position: relative;
-    top: 30%;
-    width: 70px;
-    height: 50px;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
+.friendoutcome .profile-container {
+  display: flex;
+  position: relative;
+  width: 100%;
+  height: 100%;
+  max-width: 85px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 1.625rem;
 }
 
-
-.game-records .friend-one-game-record .profile-container .profile-result-img {
-	min-width: 50px;
-    min-height: 50px;
-    background-color: #dee2e6;
-    border-radius: 50%;
-    overflow: hidden;
-    display: flex;
-    justify-content: center;
-    align-items: center;
+.friendoutcome .profile-container img {
+  max-height: 100%;
+  width: auto;
+  object-fit: contain;
 }
 
-.game-records .friend-one-game-record .profile-container .profile-result-img img {
-	width: 100%;
-	height: 100%;
-	object-fit: cover;
-	margin-left: 5px;
-	justify-content: center;
-    align-items: center;
+.friendoutcome .result {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 65px;
+  font-size: 1.875rem; /* 30px를 rem 단위로 변경 */
```

font size는 global.css를 활용해주세요. 
1.875rem을 사용하고 싶으시다면 global.css에 추가하여 변수를 언급해주세요!
```suggestion
  font-size: var(--base-font-size);
```